cqrs-postgres/src/error.rs

```diff
@@ -64,3 +64,5 @@ impl<E: CqrsError> From<postgres::Error> for LoadError<E> {
         LoadError::Postgres(err)
     }
 }
+
+// TODO: ReactionError
```

Comment:
Do we want to make an issue to track this as well (or instead of?)

Reply:
Spurious TODO (removed)